packages/client/gofer/gofer.spec

```
@@ -0,0 +1,1190 @@
+
+# Determine supported
+%if 0%{?fedora} || 0%{?rhel} >= 6
+%define with_tools 1
+%endif
+%if 0%{?fedora} || 0%{?rhel} >= 7
+%define with_systemd 1
+%endif
+%if 0%{?fedora} || 0%{?rhel} <= 7
+%define with_python2 1
+%endif
+%if 0%{?fedora} || 0%{?rhel} >= 8
+%define with_python3 1
+%endif
+%if 0%{?fedora}
+%define p2n 2
+%endif
+
+
+Name: gofer
+Version: 2.12.5
+Release: 3%{?dist}
+Summary: A lightweight, extensible python agent
+Group:   Development/Languages
+License: LGPLv2
+URL: https://github.com/jortel/gofer
+Source0: https://github.com/jortel/gofer/archive/%{name}-%{version}-1.tar.gz
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+BuildArch: noarch
+%if 0%{?with_python2}
+Requires: python%{?p2n}-%{name} = %{version}
+%else
+Requires: python3-%{name} = %{version}
+%endif
+%if 0%{?with_systemd}
+BuildRequires: systemd
+Requires(post): systemd
+Requires(preun): systemd
+Requires(postun): systemd
+%endif
+%description
+Gofer provides an extensible, light weight, universal python agent.
+The gofer core agent is a python daemon (service) that provides
+infrastructure for exposing a remote API and for running Recurring
+Actions. The APIs contributed by plug-ins are accessible by Remote
+Method Invocation (RMI). The transport for RMI is AMQP using an
+AMQP message broker. Actions are also provided by plug-ins and are
+executed at the specified interval.
+
+%prep
+%setup -q -n %{name}-%{name}-%{version}-1
+
+%build
+pushd src
+rm ./gofer/devel/test.py
+%if !0%{?with_tools}
+rm -rf ./gofer/tools/
+%endif
+%if 0%{?with_python2}
+%if 0%{?p2n}
+%{py2_build}
+%else
+%{py_build}
+%endif
+%endif
+%if 0%{?with_python3}
+%{py3_build}
+%endif
+popd
+pushd docs/man/man1
+gzip *
+popd
+
+%install
+rm -rf %{buildroot}
+pushd src
+%if 0%{?with_python2}
+%if 0%{?p2n}
+%{py2_install}
+%else
+%{py_install}
+%endif
+%endif
+%if 0%{?with_python3}
+%{py3_install}
+%endif
+popd
+
+mkdir -p %{buildroot}/usr/bin
+mkdir -p %{buildroot}/%{_sysconfdir}/%{name}
```

`%{_sysconfdir}` already includes `/`:
```suggestion
mkdir -p %{buildroot}%{_sysconfdir}/%{name}
```